app/views/shared/_login.html.erb

```diff
@@ -1,4 +1,5 @@
 <!-- ko if: window.tokenExpiresIn === null -->
+<a href="#" data-bind="click: function(){ $root.ui.sitetour.startTour(); }, visible:  $root.ui.sitetour.safePath();" class="toolbar-button">Show Tour</a>
```

Comment:
1. I am not sure if this is the right file to put the button.
2. If the tour is only gonna show up  on collections page, a function should be created in search_page and put the function name here.
3. You can put more logic in the method to check if it is on collections page so that the dependencies added on account and data_access js files can be removed.
4. ```visible``` binding hides the button for me.